### PR TITLE
Improve Debian packaging

### PR DESCRIPTION
**Pull request by strugee**
This squashes some Lintian warnings. Not all of them, but at least a decent amount :thumbsup:

### CODE REVIEW

**Comment by strugee** [comment]
Also pushed a commit making the `-dev` package depend on the binary package since that's usually what Debian library packages do.

File: debian/changelog [hunk 1]
```diff
@@ -1,4 +1,4 @@
-libtpms (0.6.0-1-dev1) RELEASED; urgency=medium
+libtpms (0.6.1-1-dev1) RELEASED; urgency=medium
```
Comment (stefanberger) [inline debian/changelog]: Can you also please add a Signed-off-by: to your patches?

File: debian/control [hunk 2]
```diff
@@ -5,7 +5,7 @@ Priority: optional
 Build-Depends: automake, autoconf, libtool, libssl-dev, sed,
  make, coreutils, pkg-config, gawk, sed, bash, dh-exec
 
-Package: libtpms
+Package: libtpms0
```
Comment (stefanberger) [inline debian/control]: With this change you also have to rename debian/libtpms libtpms.install to libtpms0.install.

Comment (stefanberger) [reply debian/control]: Are you going to make these changes? If not, I will do them myself on Thursday.

File: debian/control [hunk 3]
```diff
@@ -18,5 +18,5 @@ Package: libtpms-dev
 Architecture: all
 Section: libdevel
 Description: libtpms header files and man pages
- The libtpms-dev package provides header files and man pages for the
+ The libtpms0-dev package provides header files and man pages for the
```
Comment (stefanberger) [inline debian/control]: This is not correct.

**Comment by stefanberger** [comment]
Thanks for the PR. Though without activity on the requested changes I have to close the PR now and will make those changes myself with reference to you. Thanks.

**Pull request closed without merging**